templates/windows/nova.conf

```diff
@@ -71,3 +70,7 @@ auth_plugin = v3password
 
 [oslo_policy]
 policy_file = {{ win_dir.etc }}\policy.json
+
+## should be removed after Luci push a patch so cache to be disabled by default

```

Comment:
Here's the patch: https://review.openstack.org/#/c/615570/

Still, I'd leave this option here even after the patch gets merged. You'll need it for older branches.